test/logflare_web/controllers/source_controller_test.exs

```diff
@@ -374,13 +372,7 @@ defmodule LogflareWeb.SourceControllerTest do
     end
   end
 
-  def login_user(conn, u) do
-    conn
-    |> Plug.Test.init_test_session(%{user_id: u.id})
-    |> assign(:user, u)
-  end
-
-  def expect_user_plan(_ctx) do
+  defp expect_user_plan(_ctx) do
     expect(Logflare.Billing, :get_plan_by_user, fn _ ->
```

Comment:
can move this up to the common setup, i think. don't need separate mock setup.

Reply:
do you think we should remove this check?